### PR TITLE
[Homework #8] - JS-Advanced 1 by Ivan Ishchenko

### PR DESCRIPTION
**Pull request by ViVse**

### CODE REVIEW

File: homeworks/ivan-ishchenko/08-js-advanced-1/index.js [hunk 1]
```diff
@@ -0,0 +1,210 @@
...
+
+console.log(
+  user2.preferences.sound.maxValue === clonedUser.preferences.sound.maxValue,
+); // false
+console.log(user2.arr[2].test === clonedUser.arr[2].test); // false
+
+// task 3 https://github.com/qaprosoft/react-laba-international-2/blob/main/lectures/08-js-advanced-1/task.md#3-a-long-time-ago
+const offset = date => {
+  const today = new Date(/* '2021-02-23T14:00:00' */);
+  date = new Date(date);
```
Comment (VitalikMelnyk) [inline homeworks/ivan-ishchenko/08-js-advanced-1/index.js:83]: Bad practice to mutate params value.
Better to create a new one and work inside the function.

File: homeworks/ivan-ishchenko/08-js-advanced-1/index.js [hunk 2]
```diff
@@ -0,0 +1,210 @@
...
+    },
+  });
+}
+
+// task 9 https://www.codewars.com/kata/human-readable-time
+function humanReadable(seconds) {
+  const hours = Math.floor(seconds / 3600)
+    .toString()
+    .padStart(2, '0');
+  seconds %= 3600;
```
Comment (VitalikMelnyk) [inline homeworks/ivan-ishchenko/08-js-advanced-1/index.js:202]: Bad practice to mutate params value.
Better to create a new one and work inside the function.

**Review by VitalikMelnyk — COMMENTED** [review]
Hello @ViVse!
I have checked your PR, and left some comments.

**Comment by ViVse** [comment]
@VitalikMelnyk, fixed all issues

**Review by VitalikMelnyk — APPROVED** [review]
Well Done @ViVse!
Approved.

**Pull request merged**